src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/WebSocketResources.cs

```diff
@@ -118,4 +119,88 @@ protected override Binding GetBinding()
             return binding;
         }
     }
+
+    internal class WebSocketHttpsDuplexTextStreamedResource : EndpointResource<WSDuplexService, IWSDuplexService>
+    {
+        protected override string Protocol { get { return BaseAddressResource.Https; } }
+
+        protected override int GetPort(ResourceRequestContext context)
+        {
+            return context.BridgeConfiguration.BridgeHttpsPort;
+        }
+
+        protected override string Address { get { return "WebSocketHttpsDuplexTextStreamedResource"; } }
+
+        protected override Binding GetBinding()
+        {
+            int DefaultMaxReceivedMessageSize = 64 * 1024 * 1024;
+
```

Comment:
Nit: this kind of recurring value belongs in a Constants file.  Not required, but something to consider for cleanup